sdk/core/Azure.Core/src/Response.cs

```diff
@@ -32,17 +32,6 @@ public abstract class Response : Result, IDisposable
         /// </summary>
         public virtual ResponseHeaders Headers => new ResponseHeaders(this);
 
-        /// <summary>
-        /// Frees resources held by this <see cref="Response"/> instance.
-        /// </summary>
-        public abstract void Dispose();
-
-        /// <summary>
-        /// Indicates whether the status code of the returned response is considered
-        /// an error code.
-        /// </summary>
-        public virtual bool IsError { get; internal set; }
```

Comment:
Moved this up because we need it for protocol methods to decide whether to throw RFE or not.